contents/docs/integrate/_snippets/nextjs/app-router-setup.mdx

```diff
@@ -100,7 +100,18 @@ PostHog is now set up and ready to go. Files and components accessing PostHog on
 
 #### Capturing pageviews
 
-PostHog's `$pageview` autocapture relies on page load events. Since Next.js acts as a single-page app, this event doesn't trigger on navigation and we need to capture `$pageview` events manually. 
+There are two ways to capture pageviews:
+
+1. Use the `capture_pageview` option to capture pageviews automatically
+2. Manually capture pageviews
+
+##### Automatically capture pageviews
```

Comment:
```suggestion
##### Automatically capture pageviews (recommended)
```

Reply:
We need to make it very clear that this is the recommended and automatic way now, so maybe we shouldn't even include the previous method? Or, if we do include it, it should probably be hidden behind a fold/accordion.